crypto.go

```diff
@@ -398,6 +425,33 @@ func Sign(message []byte, key SigningKey) []byte {
 	return bytes[:offset]
 }
 
+// Verify a signature of message using the provided public key, for messages signed with TFSP1;ED25519;BLAKE2B
+func Verify(signature string, message string, publicSigningKey []byte) (string, error) {
+	if len(publicSigningKey) != PublicSigningKeySize {
```

Comment:
that way we can move (but not remove the need for) this verification logic to the constructor function, and let the verify function focus on verifying